release.py

```diff
@@ -181,6 +190,7 @@ def main():
     parser.add_argument('--skip_version_bump', default=False, action='store_true')
     parser.add_argument('--skip_latest_tag', default=False, action='store_true')
     parser.add_argument('--force', default=False, action='store_true')
+    parser.add_argument('--bump_patch_version', default=False, action='store_true')
```

Comment:
wdyt about making this a string flag `--bump_version` with options `patch`, `minor`, and `major` (find to leave `major` unimplemented for now)?

Reply:
Yeah I like that better - updated